dockge/umbrel-app.yml

```
@@ -0,0 +1,64 @@
+manifestVersion: 1
+id: dockge
+category: developer
+name: Dockge
+version: "1.4.2"
+tagline: Run custom Docker containers on your Umbrel
+description: >-
+  ⚠️ Make sure to only use named Docker volumes for your stacks and containers. Data in bind-mounted volumes
+  will be lost when the Portainer app is restarted or updated.
```

```suggestion
  will be lost when Dockge is restarted or updated.
```

I guess you used Portainer as a base (which is pretty similar) and forgot to update it here.